src/slash-bedrock/libexec/brl-which

```diff
@@ -67,6 +68,19 @@ which_bin() {
 	done
 }
 
+which_bin_all() (
+	min_args "${#}" "1"
+
+	while [ -n "${1:-}" ]; do
```

Comment:
The normal `brl which` interface doesn't really work properly with `-a`. For example, this is very ambiguous:

```
$ brl which -a cmake git
void
void-musl
arch
```

Some of the formats that leapt to mind include:

```
$ brl which -a cmake git
void: cmake
void-musl: cmake
arch: git
$ brl which -a cmake git
cmake: void
cmake: void-musl
git: arch
$ brl which -a cmake git
cmake: void void-musl
git: arch
```

Reply:
Personally
 ```
$ brl which -a cmake git
void: cmake git
void-musl: cmake
arch: git```
makes the most sens to me, as this quickly shows what has what, while still having a per distro view.